### DISCUSSION TITLE
Hitsound Additions Sampleset Keybinds

### DISCUSSION THREAD

**Discussion by EnderCraft888** [ppy/osu#30933] [Ideas] [answered]
In standard, you can use ctrl+(q, w, e, r) for the addition sampleset. I don’t know about others, but I personally use this quite often. In Lazer, it takes a while to change these addition samplesets since I must click the hitsound event, add the addition, select the box, type the full word for normal, drum, or hard, and it’s done. This is in comparison to simply double clicking the section of the object to change, ctrl + (qwer), (qwer). For the number of drum-hitfinishes I use for kick, this takes a while. I know you can’t assign this action to ctrl+(r, e) but it would be great to just have a keybind without typing required. 

If this already exists, I couldn’t find where it says so, in which case it might be helpful to have it say so somewhere in the Bank section
![IMG_6903](https://github.com/user-attachments/assets/754dc4f4-2fa6-4914-b3c8-ee3587dea33d)
^this thing

**Comment by peppy** [comment, marked as answer]
Is now added (Shift-Q~R)
